src/chan_vese.jl

```diff
@@ -0,0 +1,258 @@
+"""
+    chan_vese(img; μ, λ₁, λ₂, tol, max_iter, Δt, reinitial_flag)
```

Comment:
`[]` indicates optional argument

```suggestion
    chan_vese(img; [μ], [λ₁], [λ₂], [tol], [max_iter], [Δt], [reinitial_flag])
```